packages/docusaurus/test/load/config.test.js

```diff
@@ -13,7 +13,11 @@ describe('loadConfig', () => {
   test('website with valid siteConfig', async () => {
     const {siteDir} = await loadSetup('simple');
     const config = loadConfig(siteDir);
-    expect(config).toMatchInlineSnapshot(`
+    expect(config).toMatchInlineSnapshot(
+      {
+        plugins: expect.any(Array),
```

Comment:
The `sidebarPath` within docs plugin is now machine-dependent but I'm not sure of a good way to only allow any value for the `sidebarPath` for the docs plugin. Jest property matchers aren't that powerful to support such matching I think. I hope this is fine 😥